.gitignore

```diff
@@ -21,3 +21,6 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+.idea/
+.vscode/
```

Comment:
Trying out a couple different IDEs? I've never messed around with IntelliJ, interested to hear what you think of it.